src/DBAL/EnumType.php

```diff
@@ -81,12 +81,18 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
     /** {@inheritdoc} */
     public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
     {
-        return $platform->getVarcharTypeDeclarationSQL($fieldDeclaration);
+        return $this->name;
```

Comment:
This change will break everything if there is no type alias present in database for this type